.asf.yaml

```diff
@@ -52,5 +52,5 @@ notifications:
   # before use the config, we should ensure the "mail" address has set well (exist)
   #pullrequests_comment: issues@hugegraph.apache.org
   #issues: issues@hugegraph.apache.org
-  #discussions:  issues@hugegraph.apache.org
+  #discussions: issues@hugegraph.apache.org
```

Comment:
prefer update it in https://github.com/apache/incubator-hugegraph-toolchain/pull/345/files

Reply:
> prefer update it in [apache/incubator-hugegraph-toolchain#345 (files)](https://github.com/apache/incubator-hugegraph-toolchain/pull/345/files)

emm, how could we update server's config in toolchain's PR?

Reply:
my mistake